sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerAccessor.java

```diff
@@ -40,21 +57,91 @@
     this.batchClient = batchClient;
   }
 
-  public DatabaseClient getDatabaseClient() {
+  static SpannerAccessor create(SpannerConfig spannerConfig) {
+    SpannerOptions.Builder builder = SpannerOptions.newBuilder();
+
+    ValueProvider<Duration> commitDeadline = spannerConfig.getCommitDeadline();
+    if (commitDeadline != null && commitDeadline.get().getMillis() > 0) {
+
+      // In Spanner API version 1.21 or above, we can set the deadline / total Timeout on an API
+      // call using the following code:
+      //
+      // UnaryCallSettings.Builder commitSettings =
```

Comment:
Please file a JIRA and add a TODO for this comment linking the JIRA now that the upgrade has been done.

Your free to pick up doing this TODO in a follow-up PR.

Reply:
Added JIRA ticket in CHANGES.